README.md

```diff
@@ -43,25 +45,25 @@ Each example directory has a README.md that details how to run that specific exa
 
 0. `git clone git@github.com:looker-open-source/extension-examples.git` to clone this entire repository.
 1. `cd` to the relevant directory.
-2. `yarn install` to get all necessary packages.
-3. `yarn develop` to begin serving the bundle
+2. `npm install` to get all necessary packages.
+3. `npm run develop` to begin serving the bundle
```

Comment:
No action obv, but for posterity: we should use canonical commands like `npm start` instead of custom ones meant for the same purpose.

Reply:
I think develop came from Luke. I believe way back when it was start. I'm happy to add start in a future update but will leave develop in for those with muscle memory issues.

Reply:
Yeah, just leaving comments as notes for what we could look at doing in the future. I raised an issue to come back to these.